### PR TITLE
"enable" => "work toward"

### PR DESCRIPTION
**Pull request by cwilso**
Addresses https://github.com/w3c/AB-memberonly/issues/75.

### CODE REVIEW

**Comment by hober** [comment]
Thanks for raising this, @cwilso! FYI, @TanviHacks is on vacation right now. While @erik-anderson & I can review this before she's back, I'd like to hold off merging until she has a chance to look at it too.

**Comment by cwilso** [comment]
No problem; this isn't urgent.

**Review by hober — APPROVED** [review]
Looks good to me.

**Review by erik-anderson — APPROVED** [review]

**Review by TanviHacks — APPROVED** [review]

**Pull request merged**
